src/models/minecraft_protocol.rs

```diff
@@ -238,7 +238,7 @@ fn write_chunk_section<S: Write>(stream: &mut S, v: ChunkSection) {
     stream.write_var_int(v.data_array_length);
     let mut long: i64 = 0;
     for i in 0..4096 {
-        let block_to_place = v.block_ids[i as usize] as i64;
+        let block_to_place = i64::from(v.block_ids[i as usize]);
```

Comment:
Interesting--what difference does this make?

Reply:
Not entirely sure but it was suggested by clippy in case there is a change of variable type.